package.json

```diff
@@ -20,10 +20,9 @@
     "lodash.camelcase": "^4.3.0",
     "object-assign": "^4.0.1",
     "postcss": "^5.0.6",
-    "postcss-modules-extract-imports": "^1.0.0",
-    "postcss-modules-local-by-default": "^1.0.1",
-    "postcss-modules-scope": "^1.0.0",
-    "postcss-modules-values": "^1.1.0",
+    "postcss-icss-composes": "^2.0.3",
```

Comment:
`postcss-icss-url` seems to break a bunch of stuff for me on first testing. Might not end up being worthwhile with the `new-loader` branch being in progress. I'd still like to get https://github.com/css-modules/postcss-icss-selectors/pull/119 merged, since I think it'll be required even with `new-loader` and the reorganization of icss loaders.